### ISSUE TITLE
Unable to receive the data in Raspberry Pi

### ISSUE THREAD

**Issue by VinayVanama** [daogiang993/RPi_Controller_Bluetooth#1]
Hi, As suggested by you I have followed the steps and I have connected my application with RPI using Bluetooth. And when I'm trying to send the data to RPI I'm unable to receive it. Nothing happens after clicking the send button.

**Comment by daogiang993** [comment]
You might need to pair the phone with RPi before running anything. I used "bluetoothctl" commands.
Also, I suggest to run the RPi python code first, then launch the Android application.

**Comment by VinayVanama** [comment]
Yes I have paired my Android mobile with the RPI and it had paired successfully. And then I have executed the command which u have told in the docs and then I run the RPi_Bluetooth.py. Now I have entered data in the text box and I clicked on send button but nothing happens.

**Comment by daogiang993** [comment]
2 questions:
When you run RPi_Bluetooth.py, what is shown on the screen?
When you press CONNECT on Android phone (I haven't test with emulator), which Toast message did you get? If you get "Device not found or cannot connect", change the name in line 73. Also check in Android Studio or Eclipse to see if there is an IOException.
The first time you press SEND, wait for few second to get the socket connect because the actual socket connection is in "sendBluetoothInfo" method (my bad).

**Comment by VinayVanama** [comment]
When I executed the file RPI is showing the below screen... Also The code which u have sent doesn't have 73 lines it has only 51 line of code. I have checked for IOException also but I didnt get any. When I click send and I have waited till 5 - 10 mins and I checked but nothing happens. Also tell me one thing is the above code worked for u ? 
![whatsapp image 2017-04-13 at 11 13 05 am](https://cloud.githubusercontent.com/assets/13129801/24991859/88e2fb86-203b-11e7-9042-8cd06ed8d1f7.jpeg)
